Dockerfile

```diff
@@ -1,9 +1,15 @@
 ARG DEBIAN_VERSION=12.8
+ARG EPICS_IN_DOCKER_VERSION=v0.12.0-dev
 
-FROM ghcr.io/cnpem/lnls-debian-epics-7:v0.12.0-dev AS build-image
+FROM ghcr.io/cnpem/lnls-debian-epics-7:${EPICS_IN_DOCKER_VERSION} AS build-image
 
 FROM debian:${DEBIAN_VERSION}-slim AS base
 
+ARG DEBIAN_VERSION
+LABEL org.opencontainers.image.debian_version=${DEBIAN_VERSION}
+ARG EPICS_IN_DOCKER_VERSION
+LABEL org.opencontainers.image.epics_in_docker_version=${EPICS_IN_DOCKER_VERSION}
```

Comment:
We should have our own namespace for labels. `br.lnls. ...` or `io.github.cnpem.epics-in-docker.version`

Reply:
I use as reference the base compose.
https://github.com/cnpem/epics-in-docker/blob/a9a761cd6a33da33c20cecb5fa5ab81d21f208fa/base/docker-compose.yml#L8-L10
Do you think we should also change there in order to have consistency on using our own namespace?